plugins/modules/public_cloud_sshkey_id.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function
+
+from ansible.module_utils.basic import AnsibleModule
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+module: public_cloud_sshkey_id
+short_description: Retrieve the id for a public cloud ssh public key
+description:
+    - This module retrieves the id for a OVH public cloud public key
+author: Marco Sarti <m.sarti@onetag.com>
+requirements:
+    - ovh >= 0.5.0
+options:
+    service_name:
+        required: true
+        description: The service name
+    sshkey_name:
```

Comment:
Could you please prefix with `public_cloud` in the whole module to avoid confusion with ssh key in the `/me` route?

Reply:
Sorry for the delay, I fixed the module